.github/workflows/mysql.yml

```diff
@@ -0,0 +1,70 @@
+name: 'Build mysql docker image'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  schedule:
+    - cron: 0 7 * * THU
+  push:
+    branches:
+      - main
+    paths:
+      - mysql/**
+  pull_request:
+    branches:
+      - main
+    paths:
+      - mysql/**
+  workflow_dispatch:
+    inputs: {}
+
+jobs:
+  build-docker-image:
+    name: 'Build mysql-${{ matrix.image.version }} docker image'
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      max-parallel: 10
+      matrix:
+        image:
+          - version: 8.0
+            file: 8.0.dockerfile
```

Comment:
We should consider adapting the names of the Dockerfiles to the Docker convention: https://docs.docker.com/build/building/packaging/#filename